### DISCUSSION TITLE
Support OAuth connection type

### DISCUSSION THREAD

**Discussion by jklosinski** [Power-Maverick/DataverseDevTools-VSCode#170] [Ideas]
It would be great if the tool could support the OAuth connection type.  I'm currently not able to get administrator approval to enable the app in our tenant, and I'm rarely able to get it approved in customer tenants either.  Is there any plan to add support for OAuth connections?

**Comment by Power-Maverick** [comment]
You can use clientid and secret that uses OAuth to connect. Have you tried that?

**Reply by Power-Maverick** [reply]
Try the new version and use Microsoft Login Prompt that shouldn't require any admin approval anymore.
